src/components/elements/NavigationBar.jsx

```diff
@@ -17,17 +65,27 @@ function NavigationBar() {
 
   return (
     <nav className="border-b border-black container mx-auto lg:block">
-      <div className={styleClasses.navContainer}>
+      <div className={navStyleClasses.navContainer}>
         <div className="col-span-2">
           <Link to="/"><img src={logoImg} alt="logo"/></Link>
         </div>
 
         <div className="col-span-1"></div>
         <div className="col-span-9">
-          <div className={styleClasses.navLink}><Link to="/events/calendar">Event Calendar</Link></div>
-          <div className={styleClasses.navLink}><Link to="/events/new">Post Event</Link></div>
-          <div className={styleClasses.navLink}><Link to="/events/weekly-digest">Weekly Digest</Link></div>
-          <div className={styleClasses.navLink}>
+
+          <NavbarMenu />
+
+          <select name="navbarSupportType" component="select" className={navStyleClasses.navLink}>
```

Comment:
@victoriaklo, I think we may run into some usability issue using a select for the menu select. Let's discuss during our pairing sessions this week.